src/ActionModal/components/ActionModal.vue

```diff
@@ -457,7 +457,11 @@ export default {
       }
 
       try {
-        await this.actionManager.send(memo, feeProperties)
+        await this.actionManager.send(
+          memo,
+          feeProperties,
+          this.transactionData.validators
```

Comment:
My spidey sense tells me this should be wrapped in some kind of generic context object. There might be other properties, apart from delegates, that need to get passed into the ActionManager, and we don't want to overcrowd the parameter list.

Reply:
I'd agree. Fabo has suggested that we make a call from the extension to get the validators information to avoid a potential man in the middle attack. Doing so will eliminate this change.